helpers/multi.d.ts

```diff
@@ -2,4 +2,6 @@ export type MultiReturn<T extends any[]> = T & {
     readonly " __multiBrand": unique symbol;
 };
 
+export { MultiReturn as MR };
```

Comment:
What's the reason for adding an alias? If length is a real issue, I'd rather rename `MultiReturn` to `Multi` than see 2 different names in declarations

Reply:
If devs were to use this instead of `/** @tupleReturn */` this type would be used quite a lot especially in declarations and LuaJIT code.

If removing `/** @tupleReturn */` this type would have to be added to the end of call signatures which are usually already quite long.

I ran into a similar scenario when I was using `React.FunctionComponent` and then discovered the `React.FC` alias. Granted `FunctionComponent` is longer than `TupleReturn`.

Reply:
> If devs were to use this instead of /** @tupleReturn */ this type would be used quite a lot especially in declarations and LuaJIT code.

Yes, but it's not like `MultiReturn` is a long name. That's just 11 chars, and unlike annotations you have autocomplete there

> If removing /** @tupleReturn */ this type would have to be added to the end of call signatures which are usually already quite long.

That's only a question of formatting, i.e. you can just write it like:

```ts
declare function myFunction(
    someParameter1: Parameter1,
    someParameter2: Parameter2,
    someParameter3: Parameter3,
    someParameter4: Parameter4,
    someParameter5: Parameter5,
    someParameter6: Parameter6,
): MultiReturn<[ReturnValue1, ReturnValue2, ...OtherReturnValues[]]>
```

> I ran into a similar scenario when I was using React.FunctionComponent and then discovered the React.FC alias. Granted FunctionComponent is longer than TupleReturn.

With `React.FC` at least you have a namespace as a hint (well, in the currently recommended import style), but `MR` is pretty much impossible to understand without context

Reply:
Good points, removing this alias as it may lead to unnecessary confusion